sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -257,6 +258,28 @@ public boolean apply(SQLException e) {
     }
   }
 
+  /**
+   * This is the default {@link FluentBackoffConfiguration} that we use to retry when a {@link
+   * SQLException} occurs.
+   */
+  public static class DefaultFluentBackoffConfiguration implements FluentBackoffConfiguration {
```

Comment:
We can't make FluentBackoff part of the public API surface since it is in the `util` package. It looks like other IO connectors have been creating their own `RetryConfiguration` class such as:
SnsIO: https://github.com/apache/beam/blob/da9e17288e8473925674a4691d9e86252e67d7d7/sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/sns/SnsIO.java#L262
SolrIO: https://github.com/apache/beam/blob/da9e17288e8473925674a4691d9e86252e67d7d7/sdks/java/io/solr/src/main/java/org/apache/beam/sdk/io/solr/SolrIO.java#L225

and then converting them to any internal implementation that makes sense.

Making FluentBackoff or a RetryConfiguration public and shared across implementation might make sense but warrants a discussion on the dev@ mailing list to see what the community thinks.

Reply:
@lukecwik - cool! I will mail the dev group.

Reply:
Waiting on updates on ML thread:
https://lists.apache.org/thread.html/r7fde7b5c87c6008689a013fc113d869d3ac15cbc7a35e4534469b9ab%40%3Cdev.beam.apache.org%3E

Reply:
It looks like we should stick with the pattern other IO authors have taken in the mean time.

Reply:
I agree with @lukecwik - we should stay consistent with that.